charts/linkerd2/templates/prometheus.yaml

```diff
@@ -14,7 +14,24 @@ metadata:
   annotations:
     {{.Values.global.createdByAnnotation}}: {{default (printf "linkerd/helm %s" .Values.global.linkerdVersion) .Values.global.cliVersion}}
 data:
+  {{- if .Values.prometheusAlertingRules }}
+  alerting_rules.yml: |-
+    {{- toYaml .Values.prometheusAlertingRules | nindent 4 }}
+  {{- end }}
+
+  {{- if .Values.prometheusRecordingRules }}
+  recording_rules.yml: |-
+    {{- toYaml .Values.prometheusRecordingRules| nindent 4 }}
+  {{- end }}
+
   prometheus.yml: |-
+
```

Comment:
Extra  line ?

Reply:
The extra line caused the content to be color-coded in my IDE.
From 
![image](https://user-images.githubusercontent.com/24660299/78786819-5dcce680-7977-11ea-8267-9527237557b3.png) to 
![image](https://user-images.githubusercontent.com/24660299/78786849-67564e80-7977-11ea-89f7-ee6ba5bf0137.png)

Reply:
removed it now!